<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDialog</class>
 <widget class="QDialog" name="PatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>221</width>
     <height>219</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_id_pt">
      <property name="text">
       <string>id</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_id_pt"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_name_pt">
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_name_pt"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_gender_pt">
      <property name="text">
       <string>gender</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_tel_pt">
      <property name="text">
       <string>tel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_tel_pt"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_adrs_pt">
      <property name="text">
       <string>address</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_adrs_pt"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_birthday_pt">
      <property name="text">
       <string>birthday</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_birthday_pt"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_gender"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_family_pt">
      <property name="text">
       <string>family</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_family_pt"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_back_pt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search_pt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_info_pt">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>patients document</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search_pt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_pt">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>70</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>254</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_add_pt">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_edit_pt">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
      <property name="text">
       <string>edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_delete_pt">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
